_rules/additional-content-triggered-on-hover-is-hoverable-ep1s13.md

```diff
@@ -0,0 +1,108 @@
+rule_type: atomic
+description: |
+  This rule checks that any additional content that was triggered by hovering an element remains available to the user while the mouse pointer is over the element or the additional content
+accessibility_requirements:
+  wcag21:1.4.13: # Content on Hover or Focus (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Mouse pointer
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any element that when [hovered][] causes [visible changes in content][].
+
+## Expectation
+
+There are no [visible changes in content][] while the target element is [hovered][] or the mouse pointer remains inside a rectangle defined by the following coordinates:
+
+- (**top**:) the lowest value of the y coordinate of any pixel affected by the [visible changes in content][]; and
+- (**left**:) the lowest value of the x coordinate of any pixel affected by the [visible changes in content][]; and
+- (**bottom**:) the highest value of the y coordinate of any pixel affected by the [visible changes in content][]; and
+- (**right**:) the highest value of the x coordinate of any pixel affected by the [visible changes in content][].
+
+## Assumptions
+
+- The user does not dismiss the tooltip causing the changes in content by pressing a key on the keyboard. In this instance the rule will fail while [success criterion 1.4.13: Content on Hover or Focus][sc1.4.13] might be satisfied.
+- The content displayed on the rectangle with the changes in content does not become irrelevant. In this instance the rule will fail while [success criterion 1.4.13: Content on Hover or Focus][sc1.4.13] might be satisfied.
+- The element and the rectangle defined by the additional content are adjacent, so that it is possible to move the mouse pointer from one to the other without causing the additional content to disappear.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [Understanding Success Criterion 1.4.13: Content on Hover or Focus][sc1.4.13]
+- [F95: Failure of Success Criterion 1.4.13 due to content shown on hover not being hoverable](https://www.w3.org/WAI/WCAG21/Techniques/failures/F95)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This button element causes [visible changes in content][] by presenting a tooltip when [hovered][]. The tooltip remains displayed, resulting in no further [visible changes in content][] while the mouse pointer does not leave its boundaries or the button's boundaries.
+
+```html
+<link rel="stylesheet" type="text/css" href="/test-assets/ep1s13/styles.css" />
+<script src="/test-assets/ep1s13/scripts.js"></script>
+
+<body onload="bindEvents({tooltipRemains: true})">
+	<div class="tooltip-container">
+		<button aria-labelledby="tooltip">
+			<span aria-hidden>WCAG</span>
```

Comment:
This seems unnecessary.

```suggestion
			<span>WCAG</span>
```

Reply:
The button is labelled by the tooltip content. That's why is the span is aria-hidden.

Reply:
I agree with Wilco that it is unnecessary. The tooltip overrides the accessible name completely so there is no need to use aria-hidden to avoid duplication.

Reply:
OK. Taking it out.